corehq/messaging/smsbackends/vertex/models.py

```diff
@@ -0,0 +1,74 @@
+import urllib2, urllib
+import re
+from django.conf import settings
+from corehq.messaging.smsbackends.http.models import SQLSMSBackend
+from corehq.messaging.smsbackends.vertex.forms import VertexBackendForm
+
+VERTEX_URL = "http://www.smsjust.com/sms/user/urlsms.php"
+SUCCESS_RESPONSE_REGEX = r'^([0-9]+)-(20[0-9]{2})_([0-9]{2})_([0-9]{2})$'  # 570737298-2017_05_27
+SUCCESS_RESPONSE_REGEX_MATCHER = re.compile(SUCCESS_RESPONSE_REGEX)
+
+
+class VertexBackend(SQLSMSBackend):
+    class Meta:
+        app_label = 'sms'
+        proxy = True
+
+    @classmethod
+    def get_api_id(cls):
+        return 'VERTEX'
+
+    @classmethod
+    def get_generic_name(cls):
+        return "Vertex"
+
+    @classmethod
+    def get_available_extra_fields(self):
+        return [
+            'username',
+            'password',
+            'senderid',
+            'response',
+        ]
+
+    @classmethod
+    def get_form_class(self):
+        return VertexBackendForm
+
+    def sanitize_message(self, message):
+        # sanitize message using the rules as suggested by the api
+        # to avoid collision with reserved HTTP characters
+        return (message
+                .replace('&', 'amp;')
+                .replace('#', ';hash')
+                .replace('+', 'plus;')
+                .replace(',', 'comma;')
+                )
```

Comment:
Tested that a message is sent properly even without that sanitisation. Removing it. Thank you!